src/templates/fr/2020/methodology.html

```diff
@@ -0,0 +1,426 @@
+      </p>
+    </section>
+
+    <section>
+      <h2 id="process"><a href="#process" class="anchor-link">Processus analytique</a></h2>
+
+      <p>
+        La planification et l’exécution du Web Almanac ont pris environ un an,  avec la coordination de dizaines de contributeurs de la communauté Web.  Cette section décrit pourquoi nous avons choisi les indicateurs que vous voyez dans le Web Almanac, comment ils ont été consultés et interprétés.
+      </p>
+    </section>
+
+    <!-- To be translated <section>
+      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+
+      <p>
+        The 2020 Web Almanac kicked off in June 2020 as a <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">request for new authors</a>. After a round of brainstorming, 22 chapters were solidified and we began <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigning subject matter experts and peer reviewers to each chapter</a>. This process had some inherent bias because of the challenge of getting volunteers to commit to a project of this scale. Thus, many of the contributors are members of the same professional circles. One explicit goal for future editions of the Web Almanac is to encourage even more inclusion of underrepresented and heterogeneous voices as authors and peer reviewers.
+      </p>
+
+      <p>
+        We spent July 2020 pairing people with chapters and getting their input to finalize the individual metrics that will make up each chapter.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>
+
+      <p>
+        In July and August 2020, with the stable list of metrics and chapters, data analysts triaged the metrics for feasibility. In some cases, <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">custom metrics</a> needed to be created to fill gaps in our analytic capabilities.
+      </p>
+
+      <p>
+        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
+      </p>
+    </section>-->
```

Comment:
Can a native speaker translate this whole section and remove the `<!--` and `-->` marks?

Reply:
```suggestion
    <section>
      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">Réflexion</a></h3>

      <p>
        Le Web Almanach 2020 a été lancé en juin 2020 par un <a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">appel en direction de nouveaux auteurs et autrices</a>. Après une phase de réflexion, 22 chapitres ont été consolidés et nous avons commencé à <a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">assigner des experts et expertes sur chaque sujets et des pairs examinateurs et examinatrices à chaque chapitre</a>. Ce processus souffre d’un biais inhérent à la difficulté d’obtenir l’engagement de volontaires pour un projet de cette envergure. Ainsi, de nombreuses personnes qui contribuent au projet sont membres des mêmes cercles professionnels. Un objectif explicite des futures éditions du Web Almanac est d’encourager encore plus l’inclusion de voix sous-représentées et hétérogènes en tant qu’auteurs, autrices, relecteurs et relectrices.
      </p>

      <p>
        Nous avons passé le mois de juillet 2020 à apparier les personnes avec les chapitres et à recueillir leurs commentaires pour finaliser les indicateurs particuliers qui constitueront chaque chapitre.
      </p>
    </section>

    <section>
      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analyse</a></h3>

      <p>
        En juillet et août 2020, ayant une liste bien établie de mesures et de chapitres, les analystes de données ont trié les mesures pour en déterminer la faisabilité. Dans certains cas, il a fallu créer des <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">mesures personnalisées</a> pour combler les lacunes de nos capacités analytiques.
      </p>

      <p>
        Tout au long du mois d’août 2020, le pipeline de données HTTP Archive a parcouru plusieurs millions de sites web, rassemblant les métadonnées à utiliser dans le Web Almanac.
      </p>

      <p>
        Les analystes de données ont commencé à rédiger des requêtes pour extraire les résultats de chaque mesure. Au total, les <strong> requêtes TBC</strong> ont été écrites à la main ! Vous pouvez parcourir toutes les requêtes par chapitre dans le répertoire <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> du dépôt GitHub du projet.
      </p>
    </section>
```

Not sure I understood:

> In total, <strong>TBC queries</strong> were written by hand!

I would have thought a number would be provided.

Reply:
It will once we stop writing the queries! 😁 
though good reminder - added to #1404